sycl/doc/extensions/IntelFPGA/FPGALsu.md

```diff
@@ -0,0 +1,112 @@
+
+# FPGA lsu
+
+The Intel FPGA `lsu` class is implemented in `CL/sycl/intel/fpga_lsu.hpp` which
+is included in `CL/sycl/intel/fpga_extensions.hpp`.
+
+The class `cl::sycl::intel::lsu` allows users to explicitly request that the
+implementation of a global memory access is configured in a certain way. The
+class has two member functions, `load()` and `store()` which allow loading from
+and storing to a `global_ptr`, respectively, and is templated on the following
+4 optional paremeters:
+
+1.  **`cl::sycl::intel::burst_coalesce<N>`, where `N` is 0 or 1**: request, to
+the extent possible, that a dynamic burst coalescer be implemented when `load`
+or `store` are called.
+2. **`cl::sycl::intel::cache<N>`, where `N` is an integer greater or equal to
+0**: request, to the extent possible, that a read-only cache of the specified
+size be implemented when when `load` is called. It is not allowed to use that
+parameter for `store`.
+3. **`cl::sycl::intel::dont_statically_coalesce<N>`, where `N` is 0 or 1**:
+request, to the extent possible, that `load` or `store` accesses, should not be
+statically coalesced with other memory accesses at compile time.
+4. **`cl::sycl::intel::prefetch<N>`, where `N` is 0 or 1**: request, to the
+extent possible, that a prefetcher be implemented when `load` is called. It is
+not allowed to use that parameter for `store`.
+
+Currently, not every combination of parameters is allowed due to limitations in
+the backend. The following rules apply:
+1. For `store`, `cl::sycl::intel::cache` and `cl::sycl::intel::prefetch` must
+be `0`.
+2. For `load`, if `cl::sycl::intel::cache` is set to a value grater than `0`,
+then `cl::sycl::intel::burst_coalesce` must be set to `1`.
+3. For `load`, exactly one of `cl::sycl::intel::prefetch` and
+`cl::sycl::intel::burst_coalesce` is allowed to be `1`.
+4. For `load`, exactly only of `cl::sycl::intel::prefetch` and
```

Comment:
I'm not an English expert in any meaning, but "exactly only" looks for me not matching one to each other. "exactly one" or "the only" I suppose shall be used here.